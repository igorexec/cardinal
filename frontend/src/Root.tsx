import React from 'react';
import {App} from './components/App';

const root = () => {
  return <App />;
};

export const Root = root;
