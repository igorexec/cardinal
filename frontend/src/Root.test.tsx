import React from 'react';
import {Root} from './Root';
import {shallow, ShallowWrapper} from 'enzyme';

describe('Root', () => {
  let wrapper: ShallowWrapper;

  beforeAll(() => {
    wrapper = shallow(<Root />);
  });

  it('matches snapshot without props', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
