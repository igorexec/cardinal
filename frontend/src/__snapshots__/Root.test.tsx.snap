// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root matches snapshot without props 1`] = `<app />`;
